### PR TITLE
Preliminary UI refactoring before #9

### PR DESCRIPTION
**Pull request by n1k0**
This is preliminary refactoring of the views before starting tackling #9. 

I removed the fixed sidebar thing, which was ugly and malfunctioning. I switched the many radio buttons to be just simple dropdowns, which I'm finding quite usable and gains us a lot of real estate.

Feedback? @magopian

### CODE REVIEW

File: ui/View.elm [hunk 1]
```diff
@@ -10,85 +10,77 @@ import Types exposing (..)
...
-                ]
+filterSelector : List String -> String -> String -> (String -> Msg) -> Html Msg
+filterSelector filters filterName checkedFilter onClickHandler =
+    let
+        optionView value =
+            option [] [ text value ]
+    in
+        div [ class "form-group" ]
+            [ label [] [ text filterName ]
+            , select [ class "form-control", onInput onClickHandler, value checkedFilter ] <|
```
Comment (magopian) [inline ui/View.elm]: is that format expected? Having the `<|` on the end of a line?

Comment (n1k0) [inline ui/View.elm]: that's elm-format autoformatting

**Comment by n1k0** [comment]
I fixed a bug with the select widgets sometimes losing track of their selected value. I've also added a button to clear all filters at once, as well as rendering information about the number of matching builds and the list of filters applied:

File: ui/View.elm [hunk 2]
```diff
@@ -10,85 +10,133 @@ import Types exposing (..)
...
-                    , onClick <| onClickHandler filterValue
-                    ]
-                    []
-                , text filterValue
+            []
+        ]
+
+
+filterSelector : List String -> String -> String -> (String -> Msg) -> Html Msg
+filterSelector filters filterName checkedFilter updateHandler =
```
Comment (magopian) [inline ui/View.elm]: Now that the filters are options in a select and not radiobuttons anymore, we might want to rename `checkedFilter` to `selectedFilter`?

**Comment by n1k0** [comment]
I'm landing this.

**Comment by n1k0** [comment]
I'm gonna address your latest comments directly on master, because I'm crazy.

**Pull request merged**